__tests__/api-paths.test.js

```diff
@@ -148,7 +148,66 @@ describe("GET /api/reviews/:review_id", () => {
   });
 });
 
-describe("error handeling", () => {
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("status: 200 returns an array with the comment objects", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then((data) => {
+        const reviews = data.body.comments;
```

Comment:
this variable should be comments because that's what we're getting back from the body